app/players/elves/elf.py

```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player
+
+
+class Elf(Player):
+    def __init__(self, nickname: str, musical_instrument: str) -> None:
+        super().__init__(nickname)
+        self.musical_instrument = musical_instrument
```

Comment:
As the task requires:
`Elf` method should take an additional argument `musical_instrument` - and stored it in the **protected attribute**.
Your attribute is public, fix it.